code/modules/mob/living/silicon/robot/robot_items.dm

```diff
@@ -1,3 +1,533 @@
+////////////////////////////////////////////////////////////////////////////////
+/// Med Borg Items
+////////////////////////////////////////////////////////////////////////////////
+//Multiscanner item. Combines the functionality of the Health Analyzer, Antibody Analyzer and Advanced Reagent Scanner
+/obj/item/device/med_scanner/robot
+	name = "Med Scanner"
+	icon_state = "health"
+	item_state = "healthanalyzer"
+	desc = "An advanced integrated scanning device designed for medical analysis and diagnostics. Can quickly switch between Health, Reagent, Mass Spectrometer and Antibody Analyzer modes. Currently set to Health Analyzer mode."
+	var mode = 1
+	var scanner_upgrade = 1
+	var/list/scanners = list()
+
+	New() //1 = Robotics, 2 = Reagent, 3 = Slime
+		scanners += new /obj/item/device/healthanalyzer/advanced(src)
+		scanners += new /obj/item/device/reagent_scanner/adv(src)
+		scanners += new /obj/item/device/antibody_scanner(src)
+
+
+	attack_self(mob/user as mob)
```

Comment:
This is all relative pathed, which is directly against our contributing guidelines.
This all needs to be seperate procs,
/obj/item/device/med_scanner/robot/attack_self(mob/user as mob)